setup.cfg

```
@@ -52,7 +52,7 @@ tests_require =
   mock
   pep8-naming
   pylint
-  pytest>=3.9
+  pytest
```

Dropping `tmp_path` has relaxed this, but the way `yield` is used will still require pytest>=2.10 according to the docs: https://docs.pytest.org/en/latest/historical-notes.html#pytest-yield-fixture-decorator

This brings Fedora 29 and RHEL 8 into support, but RHEL 7 must still skip the test due to pytest 2.9.2.

```suggestion
  pytest>=2.10
```

---

How about adding the `pytest.yield_fixture` decorator instead?

---

It is deprecated in modern pytest. Your call. I'm OK with skipping the test on RHEL 7.

---

Well, it is less about "skipping the test". It is about the test dependency failing to be satisfied on RHEL 7 or Ubuntu Xenial.

---

I moved the fixture logic into the only test using it: 94f0b7c2350a82f7065cf01b8bcf7837eae646b0

---

I moved the fixture logic into the only test using it: 94f0b7c2350a82f7065cf01b8bcf7837eae646b0.

---

I'm still getting the same kind of error:
```
_____________________ ERROR at setup of test_build_package _____________________
pytest.fixture functions cannot use ``yield``. Instead write and return an inner function/generator and let the consumer call and iterate over it.:

    @pytest.fixture
    def event_loop():
        loop = new_event_loop()
        asyncio.set_event_loop(loop)
        yield loop
        loop.close()
test/test_build_python.py:49
```

---

One more fixture to roll into the test: 4842a3049c26fa92983d6339df0c97113d564415.